src/llm/http_llm_calculator.cc

```diff
@@ -51,6 +51,12 @@ enum class Endpoint {
     COMPLETIONS,
 };
 
+struct CompletionUsageStatistics {
+    size_t promptTokens = 0;
+    size_t completionTokens = 0;
+    size_t totalTokens = 0;
```

Comment:
this field is unused, remove?

Reply:
i would suggest adding method "getTotalTokens" that simply adds prompt tokens and completion tokens. this way we could re-use this code in serialization in multiple places

Reply:
I created it to make this structure in-line with OpenAI `usage` field spec, but sure I can apply your suggestion.